manifests/dev/haproxy.yaml.in

```diff
@@ -1,25 +1,15 @@
-apiVersion: v1
-kind: Service
-metadata:
-  name: haproxy
-  namespace: kube-system
-spec:
-  ports:
-    - port: 8184
-      targetPort: haproxy
-  selector:
-    app: haproxy
----
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: haproxy
   namespace: kube-system
+  labels:
+      kubevirt.io: ""
```

Comment:
Can we set labels for all resources in form `kubevirt.io/app: name` I think it can be useful if you want to see all resources for a specific name.
For example, if I want to see all resources for `haproxy` I can use `kubectl get all --all-namespaces -l "kubevirt.io/app=haproxy"`

Reply:
I am now using kubevirt.io everywhere, and where appropriate the value if it is the component.